contracts/vault.gno

```diff
@@ -0,0 +1,233 @@
+package gnoswap_optimizer
+
+import (
+	"std"
+	"strconv"
+	"time"
+
+	"gno.land/p/demo/acl"
+	"gno.land/p/demo/avl"
+	u256 "gno.land/p/gnoswap/uint256"
+	"gno.land/r/gnoswap/v2/position"
+)
+
+var (
+	vaults        *avl.Tree // tokenId -> *Vault
+	accessControl *acl.Directory
+)
+
+func init() {
+	vaults = avl.NewTree()
+	accessControl = acl.New()
+
+	accessControl.AddUserPerm(std.PrevRealm().Addr(), "role", "admin")
+}
+
+/**
+ * USER FUNCTIONS
+ */
+
+// DepositMulti deposits tokens into a vault
+// @param tokenId uint64 the vault id
+// @param _amount0 string the amount of token0 to deposit
+// @param _amount1 string the amount of token1 to deposit
+// @param _amount0Min string the minimum amount of token0 to deposit
+// @param _amount1Min string the minimum amount of token1 to deposit
+// @param deadline int64 the deadline for the transaction
+func DepositMulti(tokenId uint64, _amount0 string, _amount1 string, _amount0Min string, _amount1Min string, deadline int64) {
```

Comment:
While I get the need to write these following natspec, this doesn't follow godoc standards. You can remove `@param`

Reply:
Yes, since this format and underline is used internally within the company (onbloc), it seems better to follow the godoc style in this repository.